src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java

```diff
@@ -176,16 +188,37 @@ public void sendNotifyEmails(CaseDetails caseDetails) {
         String claimantEmailAddress = caseData.getClaimantType().getClaimantEmailAddress();
         String caseId = caseDetails.getCaseId();
 
-        if (!RESPONDENT_ONLY.equals(caseData.getSendNotificationNotify()) && !isNullOrEmpty(claimantEmailAddress)) {
-            emailService.sendEmail(sendNotificationTemplateId, claimantEmailAddress,
-                buildPersonalisation(caseDetails, emailService.getCitizenCaseLink(caseId)));
+        if (!RESPONDENT_ONLY.equals(caseData.getSendNotificationNotify())) {
+
+            if (CollectionUtils.containsAny(caseData.getSendNotificationSubject(), SendNotificationSubjects)) {
+                emailService.sendEmail(sendNotificationTemplateId,
+                        claimantEmailAddress,
+                        buildPersonalisation(caseDetails, emailService.getCitizenCaseLink(caseId)));
+            }
+
+            if (CollectionUtils.containsAny(caseData.getSendNotificationSubject(),
```

Comment:
Here and on L215, should these be else if? I'm not sure if we'd want to send more than one email even if the user has selected multiple.

Reply:
Having spoke with BA I think it is correct behaviour as hearing and other checkbox selected would send new template and if the rest of the checkbox values would send original template so in the case of for example hearing and claim(et1) were selected then both notification emails should be sent the original and the new one.